### PR TITLE
Improve accessibility for the setup wizard

### PR DESCRIPTION
**Pull request by erikjv**
Fixes: #11635

### CODE REVIEW

**Review by TheOneRing — APPROVED** [review]

**Review by TheOneRing — CHANGES_REQUESTED** [review]
During our last call we discussed a different approach, please investigate.

**Comment by TheOneRing** [comment]
> > During our last call we discussed a different approach, please investigate.
> 
> I cannot find a way to set some kind of "wizard-next-button" accessibility role. Any thoughts?

Hmm yes https://github.com/qt/qtbase/blob/0beaa63d4adf6a30c9f48919c1d9f4908de738b4/src/widgets/dialogs/qwizard.cpp#L623-L628
But I guess we should stick to the upstream text?

**Comment by erikjv** [comment]
> But I guess we should stick to the upstream text?

Done

File: src/gui/newwizard/setupwizardwindow.ui [hunk 1]
```diff
@@ -101,7 +101,7 @@
         <item>
          <widget class="QPushButton" name="errorMessageDismissButton">
           <property name="text">
-           <string>Dismiss</string>
+           <string>&amp;Dismiss</string>
```
Comment (TheOneRing) [inline src/gui/newwizard/setupwizardwindow.ui:104]: Apparently Mac doesn't like accessibility, see `Continue, `Back` and `Finish` ....
So I'm a bit lost here... 🙈

**Review by TheOneRing — APPROVED** [review]

File: src/gui/newwizard/setupwizardwindow.ui (hunk 1 above)
Comment (erikjv) [reply src/gui/newwizard/setupwizardwindow.ui:104]: Mac ignores those, they only show up on Windows and Linux.

**Pull request merged**